modules/ovirt-csi-driver-storage-class.adoc

```diff
@@ -1,30 +1,44 @@
 // Module included in the following assemblies:
 //
 // * storage/container_storage_interface/persistent-storage-csi-ovirt.adoc
+// * post_installation_configuration/storage-configuration.adoc
 
 [id="ovirt-csi-driver-storage-class_{context}"]
-= oVirt CSI driver storage class
+ifeval::["{context}" == "post-install-storage-configuration"]
+= {rh-virtualization-first} object definition
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-ovirt"]
+= {rh-virtualization-first} CSI driver storage class
+endif::[]
+
 
 {product-title} creates a default object of type `StorageClass` named `ovirt-csi-sc` which is used for creating dynamically provisioned persistent volumes.
 
 To create additional storage classes for different configurations, create and save a file with the `StorageClass` object described by the following sample YAML:
 
+.ovirt-storageclass.yaml
 [source,yaml]
 ----
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: <storage-class-name>  <1>
   annotations:
-    storageclass.kubernetes.io/is-default-class: "false"  <2>
+    storageclass.kubernetes.io/is-default-class: "<boolean>"  <2>
 provisioner: csi.ovirt.org
+allowVolumeExpansion: <boolean> <3>
+reclaimPolicy: Delete <4>
+volumeBindingMode: Immediate <5>
 parameters:
-  storageDomainName: <rhv-storage-domain-name> <3>
-  thinProvisioning: "true"  <4>
-  csi.storage.k8s.io/fstype: ext4 <5>
+  storageDomainName: <rhv-storage-domain-name> <6>
+  thinProvisioning: "<boolean>"  <7>
+  csi.storage.k8s.io/fstype: <file_system_type> <8>
 ----
 <1> Name of the storage class.
 <2> Set to `false` if the storage class is the default storage class in the cluster. If set to `true`, the existing default storage class must be edited and set to `false`.
-<3> {rh-virtualization} storage domain name to use.
-<4> Disk must be thin provisioned.
-<5> File system type to be created.
+<3> `true` enables dynamic volume expansion, `false` prevents it. `true` is recommended.
+<4> Dynamically provisioned persistent volumes of this storage class are created with this reclaim policy. This default policy is `Delete`.
+<5> Indicates how to provision and bind `PersistentVolumeClaims`. When not set, `VolumeBindingImmediate` is used. This field is only applied by servers that enable the `VolumeScheduling` feature.
+<6> The {rh-virtualization} storage domain name to use.
+<7> If `true`, the disk is thin provisioned. If `false`, the disk is preallocated. Thin provisioning is recommended.
+<8> (Optional) File system type to be created. Possible values: `ext4` (default) or `xfs`.
```

Comment:
s/(Optional) File/Optional: File
This is how we generally introduce an optional step or value (with a colon).

Reply:
Fixed. The IBM SG actually says the same thing you did.
However, I found two other instances of this in files that I don't own. I went ahead and made the changes there too, even though those files are not connected to this PR. I can undo the changes if you like.

Reply:
That's fantastic, thanks for taking care of all these.